bgpd/bgp_evpn.c

```diff
@@ -1120,7 +1120,8 @@ static int update_evpn_type5_route(struct bgp *bgp_vrf,
 	}
 
 	/* uninten temporary */
```

Comment:
should be merged with the commit related to src_attr

Reply:
This should be fine with this PR as well I guess. This PR has bug fixes and enhancements needed for symmetric routing support.